src/webserver/flick/__init__.py

```diff
@@ -0,0 +1,22 @@
+import os
+from flask import Flask
+
+def create_app():
+
+   app = Flask(__name__, instance_relative_config=True)
+
+   @app.route('/')
+   def hello_world():
+      return 'Hello World'
+
+   if __name__ == '__main__':
+      app.run(debug=True)
+
+   from . import db
```

Comment:
Make sure all the imports are at the top of the file.

Reply:
I'll change that, thanks!

Reply:
Ah, I realized that they need to be within the flask app scope, so you can leave it this way.